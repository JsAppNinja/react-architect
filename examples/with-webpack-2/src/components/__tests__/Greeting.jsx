/* eslint-env jest */
import React from 'react';
import renderer from 'react-test-renderer';

import Greeting from '../Greeting';

jest.mock('react-md/lib/Inks/InkContainer');

describe('Greeting', () => {
  it('should render correctly', () => {
    const tree = renderer.create(<Greeting />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
