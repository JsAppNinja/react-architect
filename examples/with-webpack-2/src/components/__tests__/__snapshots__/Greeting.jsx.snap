// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Greeting should render correctly 1`] = `
<div
  className="md-paper md-paper--1 md-card md-background--card greeting"
  onMouseLeave={[Function]}
  onMouseOver={[Function]}
  onTouchStart={[Function]}
>
  <div
    className="md-card-title md-card-title--primary"
    style={undefined}
  >
    <h2
      className="md-card-title--title md-text md-card-title--large"
      id={undefined}
    >
      Hello from react-md
    </h2>
  </div>
  <section
    className="md-card-text"
  >
    <p>
      This is a simple app that is using the 
      <code>
        NavigationDrawer
      </code>
       component for the layout and 
      <code>
        Card
      </code>
      , 
      <code>
        CardTitle
      </code>
      , and 
      <code>
        CardText
      </code>
       for this message.
    </p>
  </section>
  <section
    className="md-card-text"
  >
    <p>
      You can edit 
      <code>
        src/components/App.jsx
      </code>
       to start making an app. Edit 
      <code>
        src/_globals.scss
      </code>
       for any Sass variable changes.
    </p>
  </section>
</div>
`;
